INSTALLING.rst

```diff
@@ -227,7 +237,7 @@ Other option changes take effect at any time. These can be set from within
     asserts are removed. Recommended for production builds: required for any
     benchmarking.
 
-- ``ENABLE_CUDA`` - Enable compiling of the GPU accelerated computations. Default: ``OFF``.
+- ``ENABLE_HIP`` - Enable compiling of the GPU accelerated computations. Default: ``OFF``.
```

Comment:
I'm fine with using ENABLE_HIP internally in the CMake. To users though, this might be confusing as it is not a library they are required to install for CUDA support. I suggest naming the top level user option `ENABLE_GPU`. Then set ENABLE_HIP for use internally:
```
set(ENABLE_HIP ${ENABLE_GPU})
```

Reply:
done